enhancements/update/cvo-log-level-api.md

```diff
@@ -316,6 +327,29 @@ This alternative was not chosen due to it breaking the existing API consistency
 to an extent. `ClusterVersion` will continue to configure the cluster 
 version, and the new `ClusterVersionOperator` resource will configure the CVO.
 
+### The ClusterVersionOperator CRD will be applied to the management cluster
+
+As the hosted CVO is running in the management cluster, its configuration 
+resource will also be applied to the management cluster. The CRD will be 
+namespace scoped; one `ClusterVersionOperator` resource per one hosted 
```

Comment:
You'll need a namespace and cluster scoped version, which means they have slightly different requirements, and must be different resources, perhaps a ClusterVersionOperator and HostedClusterVersionOperator are both needed?

Reply:
Updated. New changes currently propose the configuration file.